conference/react-js/src/containers/Talks/index.jsx

```diff
@@ -129,8 +129,20 @@ class Talks extends Component {
     this.doFilter();
   };
 
-  sortTalks = ({ target: { tagName, dataset: { name } } }) => {
+  sortTalks = ({ target }) => {
+    const { classList, tagName, dataset: { name } } = target;
+    const currentEl = classList.value;
     if (tagName === 'TH' && name) {
+      if (currentEl.indexOf('table-header__item_active') !== -1) {
```

Comment:
there is a method `includes`. it is used to define whether string/array contains defined symbol/element.
you may use it